website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
git holds the information indeed but it's way more cumbersome to retrieve. Also if file got 50 one lines patch from 50 people git will give you 50 names that's not what you need when you need to quickly spot who is the author of the file. Adding authors should not be ambiguous for the copyright thing. The company holds the copyright indeed; it's a legal fact. The author guy if mentioned is just for knowing who to contact or hire. So I vote for keeping the current system.